_posts/2024-07-02-Basic_viewer.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "New in CGAL: Basic viewer"
+description: ""
+category:
+tags: [""]
+---
+{% include JB/setup %}
+
+<h3><a href="https://perso.liris.cnrs.fr/guillaume.damiand/">Guillaume Damiand</a>&deg;,
+Mostafa Ashraf&#42;.
+</h3>
+<h4>&deg;<a href="https://liris.cnrs.fr/">LIRIS / CNRS</a></h4>
+<br>
+
+<p>CGAL being the computational geometry algorithms library, it is often required to visualize a given data structure of the results of a certain algorithm. Since CGAL 4.13, global functions draw exist allowing to visualize (almost) all the CGAL data structures. Call a draw function opens a new interactive window showing the given model and allowing to navigate in the scene, show or hide some specific cells, show the interior of the model if any, etc. </p>
+
```

Comment:
maybe adding a minimal example code loading an obj and drawing it would be helpful to show how easy it is to display a mesh. Should be less than 10 lines